File: two-sums.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for index, num in enumerate(nums):
            i = 0
            while i < len(nums):
                
                if i==index:
                    i += 1
                    continue
                
                if nums[i] + num == target:
                    return index,i
                
                i += 1
        

File: two-sums.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int count1 = 0;
        int count2 = 0;
        vector<int> ans;
        for(auto& x : nums){
            for(auto& y : nums){
                
             if (x + y == target){
                 ans.push_back(count1);
                 ans.push_back(count2);
                 return ans;
             }   
                count2++;
            }
            count1++;
        }
        return ans;
    }
};